Sources/SwiftGRPCNIOInteroperabilityTests/protos/src/proto/grpc/testing/empty.proto

```diff
@@ -0,0 +1,28 @@
+
+// Copyright 2015 gRPC authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package grpc.testing;
+
+// An empty message that you can re-use to avoid defining duplicated empty
+// messages in your project. A typical example is to use it as argument or the
+// return value of a service API. For instance:
+//
+//   service Foo {
+//     rpc Bar (grpc.testing.Empty) returns (grpc.testing.Empty) { };
+//   };
+//
+message Empty {}
```

Comment:
Could you use `SwiftProtobuf.Google_Protobuf_Empty` instead? (Let me know if this all has just been copied over from the "official" interop tests.)

Reply:
These are copied from the official interop tests, the package naming also impacts the name of the generated type here, i.e. this becomes `Grpc_Testing_Empty` in Swift.